roles/advanced-core/advanced_dhcp_server/readme.rst

```diff
@@ -10,14 +10,14 @@ Features like shared_network, opt82, opt61 or snp/snponly roms are provided here
 Instructions
 ^^^^^^^^^^^^
 
-Please read first documentation of the standard dhcp server role. Advanced dhcp server provides same features than standard dhcp server role, but with more options.
+Please read first documentation of the standard dhcp server role. Advanced dhcp server provides sames features than standard dhcp server role, but with more options.
```

Comment:
same features was correct

Reply:
yeah, dont know Why I changed that. Correcting.